src/containers/org/components/RemoveRoleFromMemberModal.js

```diff
@@ -26,26 +26,28 @@ type Props = {
   member :any;
   onClose :() => void;
   organizationId :UUID;
-  roleId :UUID;
+  role ?:Role;
```

Comment:
Minor flow thing. The `role` prop is required, but your syntax says it's optional. Please switch to `:?Role`, which says "this prop is required by could be `null` or `undefined`. With this change, you can also get rid of the `defaultProps` below.